### DISCUSSION TITLE
History chart, only show max values

### DISCUSSION THREAD

**Discussion by tipp88** [Zefau/ioBroker.jarvis#2085] [Q&A]
Is there a setting I can do on the history chart, so that it only prints the max value of the day? 

I want to have a bar chart which shows the amount of electricy used by our heatpump per day. Currenty the bar chart gives me all values during the day and I would like to only show the last/max value. Any idea?

![image](https://user-images.githubusercontent.com/68737433/205249635-e1d41477-4761-4dd8-9b09-030a0ec7091b.png)

**Comment by Sineos** [comment]
I'm using InfluxDB and Grafana via iFrame. This gives you the utmost flexibility, e.g. 
https://user-images.githubusercontent.com/4459312/201520102-9a5fbe9e-fed2-4997-bda5-2724e444b340.png
